scania-digital-design-system/tegel
# feat(icon): enable Storybook change of title and description

**Describe pull-request**  
Improvement of SVG title and description text

**Solving issue**  
Fixes: [CDEP-3086](https://tegel.atlassian.net/browse/CDEP-3086)

**How to test**  
1. Go to the preview link
2. Change SVG title from Storybook controls 
3. Hover over the icon for a longer period
4. The same text should appear in a tooltip
5. Make text empty
6. Hover over the icon for a longer period
7. The icon name should be visible in the tooltip 

**Suggested test steps**

**Additional context**  
There are more ways to improve it, but it might be more work and we might force users to specify description prop too. If both title and descriptions are available, we might implement advice from here: [SVG A11Y](https://css-tricks.com/accessible-svgs/#aa-2-inline-svg).
Another way around is that we always specify title and description by default (probably it would be the same text) if user does not provide anything under svg-title and svg-desc props. Please let me know what you think.
